resources/views/cart/queries/cart.graphql

```diff
@@ -3,6 +3,9 @@ total_quantity
 items {
     id
     quantity
+    @if (Rapidez::checkCompadreVersion('0.0.4'))
```

Comment:
It's unfortunate that it does not match the current release but it should match the version of the module
https://github.com/rapidez/magento2-compadre/blob/master/src/etc/module.xml#L4
not the release